relay-profiling/src/sample.rs

```diff
@@ -85,6 +85,37 @@ struct Profile {
 }
 
 impl Profile {
+    /// Mutates the profile. Removes invalid samples and threads.
```

Comment:
Could we add a first short line describing the function and then the rest of the explanation, to match [relay coding guidelines (internal)](https://www.notion.so/sentry/Relay-Code-and-Architecture-4fac481cf27741aba54d39594f1cae58?pvs=4#987c0416a9f54674999eed59eecbf846)?

Reply:
Thanks did know we had this.